Server/src/client_thread.cpp

```diff
@@ -55,11 +55,12 @@ void ClientThread_cancel(void *arg) {
             delete cInfo->bmp_reader_thread;
             cInfo->bmp_reader_thread = NULL;
         }
-
+#ifndef REDIS_ENABLED
```

Comment:
>REDIS_ENABLED [](http://example.com/codeflow?start=8&length=13)

Is it possible to use runtime config instead of build time flag?

Reply:
I may prefer to use build time flag since 1> runtime config will need to introduce kafka library runtime dependency. 2> build time flag will trim unnecessary code from final binary. Now updated pipeline which will check both original kafka path and new redis path.

Reply:
If you stick with build time flag, suggest you add another build target, and keep old Kafka based Server. The repo is a openbmp fork, we should not destroy existing feature.